### PR TITLE
[IMPLEMENT] Migrated to null safety

### PR DESCRIPTION
**Pull request by Jartz**
Update Project : 
[✓] Flutter (Channel stable, 2.0.0, on macOS 11.2.1 20D74 darwin-x64, locale es-CO)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)

### CODE REVIEW

**Comment by Jartz** [comment]
@sbis04

File: example/ios/Podfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-platform :ios, '9.3'
+platform :ios, '11.0'
```
Comment (sbis04) [inline example/ios/Podfile:2]: The example project was using the LTS release. As per the [flutter_ffmpeg](https://pub.dev/packages/flutter_ffmpeg) package:

Main releases:
```
platform :ios, '11.0'
```
LTS releases:
```
platform :ios, '9.3'
```

So, the ios deployment target `9.3` should work fine.

**Review by sbis04 — CHANGES_REQUESTED** [review]
Please address the the following comments. 

Though I said that we should delay migrating the package to null safety in order to get **sound null-safety** (as two of the packages on which `video_trimmer` is dependent hasn't yet released their null-safety versions), but using other packages having higher versions with null-safety can result in version solving being failed. 

And if they contain breaking changes, then I think dependency overriding also might fail.

Ref: #92

**Comment by sbis04** [comment]
Have migrated the package to null safety, thanks for the contribution!

**Pull request closed without merging**